### PR TITLE
Personal info user page and fixing breadcrumb routes to /personal  route

### LINKED ISSUE #45: create personal information page

### PR DESCRIPTION
**Pull request by AdanRL**
fix #45

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -37,6 +38,7 @@ function App() {
           <Route path="/signup" element={<SignUpPage />} />
           <Route path="/login" element={<LogInPage />} />
           <Route path="*" element={<Page404 />} />
+          <Route path="/personal" element={<PersonalPage/>} />
```
Comment (elenamsd) [inline src/App.tsx:41]:
```suggestion
          <Route path="/personal" element={ userLogged !== "" ? <PersonalPage/> : <WebPage />} />
```

File: src/components/InputForm/InputForm.tsx [hunk 2]
```diff
@@ -16,10 +18,16 @@ const pStyle: CSSProperties = {
...
+const FormControlStyled = styled(FormControl)(() => ({
+  width: "90%",
+}));
+
 export const InputForm: FC<Props> = ({ name, type, placeholder, title="lexa", onChange, validation=false, isRequired=false }) => {  
   const [isActive, setActive] = useState<boolean>(false);
   const [message, setMessage] = useState<string>("");
+  const userLogged = useAppSelector((state) => state.user);
 
+  console.log(userLogged);
```
Comment (elenamsd) [inline src/components/InputForm/InputForm.tsx:30]:
```suggestion
```

File: src/components/InputForm/InputForm.tsx [hunk 3]
```diff
@@ -43,10 +51,10 @@ export const InputForm: FC<Props> = ({ name, type, placeholder, title="lexa", on
   }; 
 
   return (
-    <FormControl variant="filled" error={message !== ""} margin="normal">
+    <FormControlStyled variant="filled" error={message !== ""} margin="normal">
       <InputLabel>{title}{ isRequired ? "*" : null }</InputLabel>
-      <Input onChange={handleMessageChange} name={name} placeholder={placeholder} type={type} required={isRequired}/>
+      <Input id="input-with-sx" onChange={handleMessageChange} name={name} placeholder={placeholder} type={type} required={isRequired}/>
```
Comment (elenamsd) [inline src/components/InputForm/InputForm.tsx:56]:
```suggestion
      <Input onChange={handleMessageChange} name={name} placeholder={placeholder} type={type} required={isRequired}/>
```

File: src/components/Menu/Menu.tsx [hunk 4]
```diff
@@ -17,15 +17,15 @@ export const SideMenu: FC = () => {
   
   const items = [
     { icon: <Home color="info" />, name: "Home", separator: <ListSubheader>Personal</ListSubheader> },
-    { icon: <Event color="info" />, name: "Calendar" },
-    { icon: <Settings color="info" />, name: "Settings" },
-    { icon: <SupervisedUserCircle color="info" />, name: "Nutritionists", separator: <ListSubheader>Services</ListSubheader> },
-    { icon: <MenuBook color="info" />, name: "Recipes" },
+    { icon: <Event color="info" />, name: "Calendario" },
+    { icon: <Settings color="info" />, name: "Configuración" },
+    { icon: <SupervisedUserCircle color="info" />, name: "Nutritionistas", separator: <ListSubheader>Servicios</ListSubheader> },
```
Comment (elenamsd) [inline src/components/Menu/Menu.tsx:22]:
```suggestion
    { icon: <SupervisedUserCircle color="info" />, name: "Nutricionistas", separator: <ListSubheader>Servicios</ListSubheader> },
```

File: src/components/PersonalForm/PersonalForm.tsx [hunk 5]
```diff
@@ -1,13 +1,49 @@
...
+import SaveIcon from "@mui/icons-material/Save";
+import CancelIcon from "@mui/icons-material/Cancel";
+import { styled } from "@mui/material/styles";
+
 import { InputForm } from "../InputForm/InputForm";
 
+const FormContainer: CSSProperties = {
+  display: "flex",
+  flexGrow: 1,
+  // width: "100vh",
```
Comment (elenamsd) [inline src/components/PersonalForm/PersonalForm.tsx:16]:
```suggestion
```

File: src/components/PersonalForm/PersonalForm.tsx [hunk 6]
```diff
@@ -1,13 +1,49 @@
...
+  // height: "70vh",
```
Comment (elenamsd) [inline src/components/PersonalForm/PersonalForm.tsx:17]:
```suggestion
```

File: src/components/PersonalForm/PersonalForm.tsx [hunk 7]
```diff
@@ -1,13 +1,49 @@
...
+
 export const PersonalForm: FC = () => {
   const [data, setData] = useState({
-    username: "adan",
-    email: "axeltaj@gmail.com",
-    phone: 35711533,
-    Descripcion: "Especialista en nutrición deportiva, ¿necesitas consejo?,¡llamame! <3"
+    name: "",
+    phone: 0,
+    Descripcion: ""
```
Comment (elenamsd) [inline src/components/PersonalForm/PersonalForm.tsx:46]: should it be in english? description?

**Review by elenamsd — CHANGES_REQUESTED** [review]

File: src/components/PersonalForm/PersonalForm.tsx (hunk 7 above)
Comment (AdanRL) [reply src/components/PersonalForm/PersonalForm.tsx:44]: should be empty

File: src/components/InputForm/InputForm.tsx [hunk 8]
```diff
@@ -16,9 +18,14 @@ const pStyle: CSSProperties = {
   color: "red"
```
Comment (mnzgarcia) [inline src/components/InputForm/InputForm.tsx:18]:
```suggestion
  color: "red";
```

File: src/components/PersonalForm/PersonalForm.tsx [hunk 9]
```diff
@@ -1,13 +1,47 @@
...
```
Comment (mnzgarcia) [inline src/components/PersonalForm/PersonalForm.tsx:44]:
```suggestion
    description: ""
```

**Review by mnzgarcia — CHANGES_REQUESTED** [review]

**Review by elenamsd — APPROVED** [review]
😍

**Review by mnzgarcia — APPROVED** [review]

**Pull request merged**